pkg/controller.v1/pytorch/job_test.go

```diff
@@ -238,6 +238,46 @@ func TestDeletePodsAndServices(t *testing.T) {
 			expectedPodDeletions: 5,
 			expectedServiceDeletions: 1,
 		},
+		testCase{
+			description: "4 workers and 1 master running, policy is running",
+			job: 		 testutil.NewPyTorchJobWithCleanPolicy(1, 4, common.CleanPodPolicyRunning),
+
+			pendingWorkerPods:   0,
+			activeWorkerPods:    4,
+			succeededWorkerPods: 0,
+			failedWorkerPods:    0,
+
+			pendingMasterPods:   0,
+			activeMasterPods:    1,
+			succeededMasterPods: 0,
+			failedMasterPods:    0,
+
+			activeWorkerServices: 4,
+			activeMasterServices: 1,
+
+			expectedPodDeletions: 5,
+			expectedServiceDeletions: 1,
+		},
+		testCase{
+			description: "4 workers and 1 master succeeded, policy is running",
+			job: 		 testutil.NewPyTorchJobWithCleanPolicy(1, 4, common.CleanPodPolicyRunning),
```

Comment:
Please use gofmt to format the code.

Reply:
execute command: "gofmt -l -w  ./pkg/controller.v1/pytorch/job_test.go"